File: modul-04/bahasa c/PRAK402-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

int main()
{
    int Min = 1;
    int Max;
    scanf("%d", &Max);

    for (int a = Min; a <= Max; a++)
    {
        if (a % 2 != 0)
        {
            printf("%d ", a);
        }
    }

    printf("\n");

    for (int a = Max; a >= Min; a--)
    {
        if (a % 2 == 0)
        {
            printf("%d ", a);
        }
    }

    printf("\n");
    return 0;
}

File: modul-04/python/PRAK402-2310817310003-Mfaridsetiawan.py
Min = 1
Max = int(input(""))

for a in range(Min , Max +1):
  if a % 2 == 0:
   print('' , end='')
  else :
    print(a,'', end='')

print('')

for a in range(Max,Min, -1):
  if a % 2 == 0:
    print(a,'', end='')